cmd.go

```diff
@@ -189,9 +191,10 @@ func OptDetails(details string) optionFn {
 func New(options ...optionRoot) *Cmd {
 	// Set default config.
 	cfg := config{
-		name:          os.Args[0],
-		errorHandling: flag.ExitOnError,
-		output:        os.Stderr,
+		name:              os.Args[0],
+		errorHandling:     flag.ExitOnError,
+		output:            os.Stderr,
+		completionEnabled: detectCompletionSupport(),
```

Comment:
I don't think it is necessary to store this value in the config.
You can just use the function to calculate it on the fly.

Reply:
Done 👍